pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: fl_chart
 description: A highly customizable Flutter chart library that supports Line Chart, Bar Chart, Pie Chart, Scatter Chart, and Radar Chart.
-version: 0.70.2
+version: 0.70.3
```

Comment:
You also don't need to update the version. I will do it when we have enough changes. Please read our guideline [here](https://github.com/imaNNeo/fl_chart/blob/main/CONTRIBUTING.md)